projects/com.oracle.truffle.llvm.parser.impl/src/com/oracle/truffle/llvm/parser/impl/LLVMVisitor.java

```diff
@@ -672,6 +672,9 @@ private LLVMNode visitFunctionCall(Callee callee, EList<Argument> args, Resolved
             String functionName = functionHeader.getName();
             if (functionName.startsWith("@llvm.")) {
                 return factoryFacade.createLLVMIntrinsic(functionName, finalArgs, containingFunctionDef);
+            } else if (functionName.equals("@truffle_get_arg")) {
+                assert finalArgs.length == 2 : finalArgs.length;
+                return factoryFacade.createTruffleGetArgIntrinsic(finalArgs[finalArgs.length - 1]);
```

Comment:
Maybe we can have a `createTruffleIntrinsic` instead of `createTruffleGetArgIntrinsic` in the facade? For now, you could just handle `@truffle_get_arg` there and return `null` in the other cases. If the method returns `null` you can just construct a normal function call node (which is resolved during runtime in the `LLVMRuntimeIntrinsicFactory`). Then, we could also migrate the other Truffle intrinsics from `LLVMRuntimeIntrinsicFactory` in a separate step.